templates/bundles/PrestaShopBundle/Admin/Sell/Catalog/Product/Form/categories_form_theme.html.twig

```diff
@@ -24,49 +24,80 @@
  *#}
 {% extends '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig' %}
 
-{% block product_category_row %}
-  <li class="category-tree-element">
-    <div class="checkbox category-tree-inputs">
-      {{ form_widget(form.is_associated) }}
-      {{ form_widget(form.is_default) }}
-    </div>
-    <ul class="children-list"></ul>
-  </li>
+{% block categories_row %}
```

Comment:
I'm not sure this one is really needed anymore, it looks a lot like a classic `form_row` If you can remove this one it would allow modules to append other fields into the CategoryType if needed (with this hard coded override it is not possible)